tests/formats/rewards/README.md

```diff
@@ -0,0 +1,47 @@
+# Rewards tests
+
+The different rewards deltas sub-functions are testing individually with the test handlers, each returning the related `rewards`/`penalties`.
+There is no "change" factor, the rewards/penalties outputs are pure functions with just the pre-state as input.
+Hence, the format is shared between each test-handler. (See test condition documentation on how to run the tests.)
+
+
+## Test case format
+
+### `meta.yaml`
+
+```yaml
+description: string    -- Optional description of test case, purely for debugging purposes.
+                          Tests should use the directory name of the test case as identifier, not the description.
+bls_setting: int       -- see general test-format spec.
+```
+
+### `pre.yaml`
+
+A YAML-encoded `BeaconState`, the state before running the rewards sub-function.
+
+Also available as `pre.ssz`.
+
+
+### `rewards.yaml`
+
+A YAML-encoded list of integers representing the 0th item in the return value (i.e. the rewards deltas)
```

Comment:
TODO: update to match ssz format optionality. Also since yaml doesn't work for all clients, and converting to json causes more integer encoding problems for clients to deal with.